### ISSUE TITLE
[Bug]: Date input is missing an aria-label

### ISSUE THREAD

**Issue by EmmaDawsonDev** [schmelto/LinkFree_generator#17]
### What happened?

The date input element in the milestones section has no label and therefore needs an aria-label attribute adding to make it accessible.

`<input type="date" name="date" aria-label="date" class="form-control" placeholder="">`

### What browsers are you seeing the problem on?

Chrome
